uw-frame-components/staticFeeds/features_template.json

```diff
@@ -16,16 +16,14 @@
       "startMonth": 7,
       "startDay": 11,
       "endYear": 2015,
-      "endMonth": 7,
+      "endMonth": 7, (0 based index)
```

Comment:
Ah, hadn't caught that it was a template.

:+1: 

Moving this into the documentation sounds like a good idea, but I don't think it needs to block this bit of inline documentation.